src/api/entities/Identity/index.ts

```diff
@@ -800,4 +812,42 @@ export class Identity extends Entity<UniqueIdentifiers, string> {
       middlewareInstructionToHistoricInstruction(instruction!, context)
     );
   }
+
+  /**
+   * Returns the list of all child identities
+   *
+   * @note this query can be potentially **SLOW** depending on the number of parent Identities present on the chain
```

Comment:
```suggestion
   * @note this query can be potentially slow depending on the number of parent Identities present on the chain
```

I am not sure if we should even mention this. Hopefully we have a better chain method before its an issue, and its not like developers can do much other than complain and us add to SubQuery or ask core to add a storage value.

Reply:
I guess we could monitor with a cronjob, like:

```ts
const start = new Date()
await identity.getChildIdentities()

if(new Date() - start > 1000) postToSlack()
```

Reply:
I personally didn't wanted to add the comment but it was discussed in the call to add in to explicitly specify the slowness so here it is. Also, the slow comment was mentioned the same way somewhere else that's why I followed the caps bold pattern for **SLOW**. Also, where do we add the cron?